lcls-twincat-motion/Library/DUTs/ST_AxisParameterSetExposed.TcDUT

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1">
+  <DUT Name="ST_AxisParameterSetExposed" Id="{ee296551-543a-4bbb-98b7-123ff2229439}">
+    <Declaration><![CDATA[TYPE ST_AxisParameterSetExposed :
+    // Collects the axis parameters that are intentionally exposed to EPICS.
+STRUCT
+    // Maximum Dynamics
+    {attribute 'pytmc' := '
+        pv: MaxVel
+        io: i
+        field: DESC Maximum commandable speed of the axis in EU/s.
+    '}
+    fVeloMaximum                : LREAL; // Maximum commandable speed of the axis in EU/s.
+    {attribute 'pytmc' := '
+        pv: MaxAccel
+        io: i
+        field: DESC Maximum rate of increase in speed of the axis in EU/s^2.
+    '}
+    fAccelerationMax            : LREAL; // Maximum rate of increase in speed of the axis in EU/s^2.
+    {attribute 'pytmc' := '
+        pv: MaxDecel
+        io: i
+        field: DESC Maximum rate of decrease in speed of the axis in EU/s^2.
+    '}
+    fDecelerationMax            : LREAL; // Maximum rate of decrease in speed of the axis in EU/s^2.
+
+    // Monitoring
+    {attribute 'pytmc' := '
+        pv: PosLagEn
+        io: i
+        field: DESC TRUE if position lag monitor (also known as stall monitor) is enabled.
+    '}
+    bCtrlEnablePosDiffControl   : WORD;  // Enable/Disable state of Position Lag Monitor.
+    {attribute 'pytmc' := '
+        pv: PosLagVal
+        io: i
+        field: DESC Maximum magnitude of position lag in EU.
+    '}
+    fCtrlPosDiffMax             : LREAL; // Maximum magnitude of position lag in EU.
+    {attribute 'pytmc' := '
+        pv: PosLagTime
```

Comment:
This is the longest new PV name and it's still shorter than the longest existing PV so we're good on PV lengths here :+1:
`:PLC:AxisPara:PosLagTime`
vs
`:PLC:bGantryBackwardEnable`

Reply:
Note for future: here's how I found the longest as-deployed PV name, with a big assist from stackoverflow for use of awk:
```
(pcds-5.8.2)zlentz@psbuild-rhel7-01:~$ grep_pv :PLC: | grep MMS | awk ' { if ( length > x ) { x = length; y = $0 } }END{ print y }'
SP1L0:KMONO:MMS:DIODE_HORIZ:PLC:bGantryBackwardEnable_RBV, "bi"
```